apps/www/app/templates/data.ts

```diff
@@ -1,22 +1,21 @@
 type StrArrayToUnion<TArr extends readonly string[]> = TArr[number];
 
-// sort these alphabetically
```

Comment:
It's unrealistic to expect everyone going forward to sort these correctly. It should be sorted dynamically in the page itself